README.md

```diff
@@ -94,11 +96,11 @@ The race detector can be enabled by setting `XK6_RACE_DETECTOR=1`.
 
 ```go
 builder := xk6.Builder{
-	k6Version: "v2.0.0",
+	k6Version: "v0.29.0",
 	Plugins: []xk6.Dependency{
```

Comment:
There's a lot of mention of "plugins" still, but not sure if we should bother with converting them all to "extensions".

Reply:
I am for `s/plugin/extension/`

Reply:
Agreed, let's do this in #7.